js/src/common/Application.js

```diff
@@ -81,6 +82,7 @@ export default class Application {
    * @public
    */
   store = new Store({
+    'access-tokens': AccessToken,
```

Comment:
Isn't the convention for JS variables and attributes to use camelCase?

Reply:
I think this is `access-tokens` because of the `type` field in the serializer.

Reply:
Yes that's the type, and the prefix is because there are multiple kinds of tokens (see the other comment). None of the other tokens have a serializer (yet) though.

It will be the first two-word serializer type used in Flarum core, so we have some room on how to name them. In my extensions I have usually used the dash-syntax for such types, like `catchthefish-fishes`, but of course we could also use camelCase.

Reply:
I think camelCase would be best here - I have used hyphens in the past as well, and usually am confused as to how I should be naming them.

Reply:
The JSON:API spec does not make any recommendation on this https://jsonapi.org/format/#document-resource-object-identification

**However**, the type is usually matched to the url endpoint. I don't really like the idea of having a camelCase endpoint url...

Reply:
I don't think it should match the URL endpoint. Everything returned by the API right now is camelCase (model attributes), so adding hyphens really doesn't make sense.

Reply:
Agree with David's point. But if we're going for word separators in an endpoint, I believe that snake case is the convention?

Reply:
We do already have a two-word serializer, type `mail-settings`. Let's just stick to that, everything else seems like bikeshedding to me. :wink: